recipes/mdonatello/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mdonatello" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mdonatello-{{ version }}.tar.gz
+  sha256: 719784cb4fc8a64a4df1aef484eac7e6220a142ea2b39db90e87056459b097d8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools >=61.2
+  run:
+    - python >=3.9
+    - MDAnalysis >=2.0.0
+    - jupyter
```

Comment:
Should this be `jupyter` or:

```
- notebook
```
?

Reply:
notebook would be enough for this package, adjusted it.